Winapp2.ini

```diff
@@ -937,125 +875,131 @@ RegKey1=HKLM\Software\FullCircle\TalkBack
 [Telemetry *]
 LangSecRef=3026
 SpecialDetect=DET_MOZILLA
+DetectFile1=%AppData%\Moonchild Productions\Pale Moon
+DetectFile2=%AppData%\Mozilla\SeaMonkey
+DetectFile3=%AppData%\Waterfox
 Default=False
-FileKey1=%AppData%\Mozilla\FireFox\Profiles\*|Telemetry*.*
-FileKey2=%AppData%\Mozilla\Firefox\Profiles\*\datareporting\archived|*.jsonlz4|REMOVESELF
-FileKey3=%AppData%\Mozilla\FireFox\Profiles\*\saved-telemetry-pings|*.*
-FileKey4=%LocalAppData%\Mozilla\Firefox\Profiles\*|ShutdownDuration.*
+FileKey1=%AppData%\Moonchild Productions\Pale Moon\Profiles\*|Telemetry*.*
+FileKey2=%AppData%\Moonchild Productions\Pale Moon\Profiles\*\datareporting\archived|*.jsonlz4|RECURSE
+FileKey3=%AppData%\Moonchild Productions\Pale Moon\Profiles\*\saved-telemetry-pings|*.*
+FileKey4=%AppData%\Mozilla\Firefox\Profiles\*|Telemetry*.*
+FileKey5=%AppData%\Mozilla\Firefox\Profiles\*\datareporting\archived|*.jsonlz4|RECURSE
+FileKey6=%AppData%\Mozilla\Firefox\Profiles\*\saved-telemetry-pings|*.*
+FileKey7=%AppData%\Mozilla\SeaMonkey\Profiles\*|Telemetry*.*
+FileKey8=%AppData%\Mozilla\SeaMonkey\Profiles\*\datareporting\archived|*.jsonlz4|RECURSE
+FileKey9=%AppData%\Mozilla\SeaMonkey\Profiles\*\saved-telemetry-pings|*.*
+FileKey10=%AppData%\Waterfox\Profiles\*|Telemetry*.*
+FileKey11=%AppData%\Waterfox\Profiles\*\datareporting\archived|*.jsonlz4|RECURSE
+FileKey12=%AppData%\Waterfox\Profiles\*\saved-telemetry-pings|*.*
+FileKey13=%LocalAppData%\Moonchild Productions\Pale Moon\Profiles\*|ShutdownDuration.*
+FileKey14=%LocalAppData%\Mozilla\Firefox\Profiles\*|ShutdownDuration.*
+FileKey15=%LocalAppData%\Mozilla\SeaMonkey\Profiles\*|ShutdownDuration.*
+FileKey16=%LocalAppData%\Waterfox\Profiles\*|ShutdownDuration.*
 
 [Temps *]
 LangSecRef=3026
 SpecialDetect=DET_MOZILLA
 Default=False
-FileKey1=%LocalAppData%\VirtualStore\Program Files*\Mozilla*|*.tmp|RECURSE
```

Comment:
Why was this line removed? Does the system not create this directory when the user account control is enabled?

Reply:
I removed the few VirtualStore lines because I never had any files or folders under VirtualStore (on all my Windows 7, 8.1 and 10 systems). My UAC is always set to the highest level.

IMHO most of the VirtualStore lines aren't necessary. Only very old applications (coded for Windows XP and older) are redirected to VirtualStore.

Reply:
So this isn't valid for version 52esr for XP users?

Reply:
No, because Windows XP doesn't have the VirtualStore feature at all. This was introduced with Windows Vista for better compatibility with the new UAC.

Firefox was fully compatible with Windows Vista and higher at all time. Therefore, VirtualStore isn't used.

Reply:
Thanks for the clarification :)